app/models/miq_region.rb

```diff
@@ -289,6 +293,15 @@ def self.display_name(number = 1)
     n_('Region', 'Regions', number)
   end
 
+  def reload(*_, &block)
```

Comment:
I'm surprised this is the only way to "hook" reload...though there would be a more elegant way :(